package.json

```diff
@@ -1,13 +1,13 @@
 {
   "private": true,
   "scripts": {
-    "clean": "rimraf output && rimraf .pulp-cache",
-    "build": "pulp build -- --censor-lib --strict",
-    "test": "pulp test"
+    "clean": "rimraf output",
+    "build": "spago build",
+    "test": "spago -x test/test.dhall test"
```

Comment:
There's a separate config for tests so that test dependencies can be kept out of the generated Bower file.